modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/provider/JournalArticleAnalyticsReportsInfoItemObjectProvider.java

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: